packages/relay/src/api/index.ts

```diff
@@ -10,10 +10,75 @@ import {
   SignedMessagePayload,
   SignTypedDataPayload,
   SignMessagePayload,
+  CreateRelayerRequest,
+  RelayerListResponse,
+  UpdateRelayerPoliciesRequest,
+  UpdateRelayerRequest,
 } from '../relayer';
 
 export const RelayerApiUrl = () => process.env.DEFENDER_RELAY_API_URL || 'https://api.defender.openzeppelin.com/';
 
+export class RelayClient extends BaseApiClient {
+  protected getPoolId(): string {
+    return process.env.DEFENDER_RELAY_CLIENT_POOL_ID || 'us-west-2_94f3puJWv';
+  }
+
+  protected getPoolClientId(): string {
+    return process.env.DEFENDER_RELAY_CLIENT_POOL_CLIENT_ID || '40e58hbc7pktmnp9i26hh5nsav';
+  }
+
+  protected getApiUrl(): string {
+    return process.env.DEFENDER_RELAY_CLIENT_API_URL || 'https://defender-api.openzeppelin.com/';
+  }
+
+  public async get(relayerId: string): Promise<RelayerModel> {
+    return this.apiCall(async (api) => {
+      return await api.get(`/relayers/${relayerId}`);
+    });
+  }
+
+  public async list(): Promise<RelayerListResponse> {
+    return this.apiCall(async (api) => {
+      return await api.get('/relayers/summary');
+    });
+  }
+
+  public async create(relayer: CreateRelayerRequest): Promise<RelayerModel> {
+    return this.apiCall(async (api) => {
+      return await api.post('/relayers', relayer);
+    });
+  }
+
+  public async update(relayerId: string, relayerUpdateParams: UpdateRelayerRequest): Promise<RelayerModel> {
```

Comment:
As a consumer of this package, it would surprise me if `update` did not update all relevant fields I included in `relayerUpdateParams` so I abstracted away the detail of how these API calls happen into this single function. We could potentially also expose `updatePolicies` if we think that would be of value.

Reply:
Makes complete sense to me. Actually this is something I would've pushed for. Thanks for bringing it up and worry for consistency 🙌